Products/CMFPlone/controlpanel/browser/usergroups_groupmembership.py

```diff
@@ -92,7 +92,7 @@ def getMembers(self):
             'fullname') is not None and normalizeString(x.getProperty('fullname')) or '')
 
         mergedResults = groupResults + userResults
-        return filter(None, mergedResults)
+        return [i for i in filter(None, mergedResults)]
```

Comment:
this is completely wrong, this must be replaced by:

```python
[i for i in mergedResults if i]
```

Reply:
@hvelarde True. You are welcome to fixing it.

Reply:
@pbauer done! :)